src/common/transformations/include/transformations/utils/utils.hpp

```diff
@@ -61,19 +61,8 @@ inline bool has_decompression_converts(const std::shared_ptr<const ov::Model>& f
 }
 
 inline std::string create_ie_output_name(const Output<const Node>& output) {
-    std::string out_name;
-    OPENVINO_SUPPRESS_DEPRECATED_START
-    auto tensor_name = ov::descriptor::get_ov_tensor_legacy_name(output.get_tensor());
-    OPENVINO_SUPPRESS_DEPRECATED_END
-    if (!tensor_name.empty()) {
-        out_name = std::move(tensor_name);
-    } else {
-        const auto& prev_layer = output.get_node_shared_ptr();
-        out_name = prev_layer->get_friendly_name();
-        if (prev_layer->get_output_size() != 1) {
-            out_name += "." + std::to_string(output.get_index());
-        }
-    }
+    const auto& prev_layer = output.get_node_shared_ptr();
+    std::string out_name = prev_layer->get_friendly_name();
     return out_name;
```

Comment:
This change will using friendly_name for output name, is it make sense?

Reply:
It seems legit since we get rid of the legacy part. However, can we just drop the create_ie_output_name? it looks like this method is private and we can delete it without deprecation